gobblin-metrics-libs/gobblin-metrics-base/src/main/java/gobblin/metrics/event/sla/SlaEventSubmitter.java

```diff
@@ -57,6 +57,9 @@
   private String dedupeStatus;
   private String completenessPercentage;
   private String isFirstPublish;
+  private String sourceCluster;
```

Comment:
I don't think this should be top-level fields of `SlaEventSubmitter`, as they don't make sense for all datasets. Instead put them in additionalMetadata.